### PR TITLE
Removes some special cases for the chaplains religion

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Removes the special case for homosexuality and some shitty ones.
Choosing this will get you bwoinked anyway so why encourage it with a special name?

**Changelog:**
:cl:
del: Removes some of the chaplain special religion names
/:cl:

### CODE REVIEW

**Comment by FalseIncarnate** [comment]
I feel like removing this from causing near-fatal brain damage would be actually encouraging this more than the “special name” does actually does.

For reference, 100 brain damage is pretty much instant Crit/death. This puts you at 99. Bible healing deals 10, and at this level the chaplain is likely to head butt his own door trying to get out and can’t reliably use any machine. He’s going to likely die before he can get treatment because of his own literal stupidity.

**Comment by taukausanake** [comment]
Yeah, I think picking it is itself punishment enough. You aren't going to live long enough.

Do people actually get bwoinked for picking that? I figured it would have been removed long before if that was the case

File: code/game/jobs/job/support_chaplain.dm [hunk 1]
```diff
@@ -64,11 +64,6 @@
 				B.name = "Uplifting Primer"
 			if("toolboxia")
 				B.name = "Toolbox Manifesto Robusto"
-			if("homosexuality")
-				B.name = "Guys Gone Wild"
-			if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks")
-				B.name = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition")
-				H.setBrainLoss(99) // starts off retarded as fuck
```
Comment (Fox-McCloud) [inline code/game/jobs/job/support_chaplain.dm]: In my opinion, the latter one should stay, though the name of the books should probably be changed, along with potentially a few of the descriptors to get it.

Comment (farie82) [reply code/game/jobs/job/support_chaplain.dm]: Making them get braindamage is still a reward for them. I'm fine either way to be honest. Currently out of creativity for names or such so I'm open for suggestions :D

**Pull request merged**